### PR TITLE
Avoid docker login everytime even it is done before

### PR DESCRIPTION
**Pull request by LinuxSuRen**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Make it becomes more convenience when we build docker image.

**Which issue(s) this PR fixes**:
None.

**Special notes for reviewers**:

**Additional documentation, usage docs, etc.**:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -79,6 +79,8 @@ openapi:
 # Build the docker image
 docker-build: all
 	hack/docker_build.sh
+docker-build-no-test: ks-apiserver controller-manager
+	hack/docker_build.sh
```
Comment (shaowenchen) [inline Makefile:83]: What cases do we need to skip unit tests?

Comment (LinuxSuRen) [reply Makefile:83]: It works well when just need to make a docker image on a personal computer quickly. Of course, it's not a good practice for using it.

**Pull request merged**